### PR TITLE
Kick Off @GSoC2020

### PR DESCRIPTION
**Pull request by fahad-israr**
- Added Starfish Description
- Workflow
- Necessary Utilities
- Targeted Websites
- Target Platform
- Technology for Development

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,50 +1,31 @@
-# Project Starfish
+# Project Starfish 
 
-GSOC 2020 Starfish POC
+A  cross platform client side application that would let you open a file in IDE of your choice (vscode, eclipse, intellij, emacs, vi, etc.) or start a debugger locally directly from the browser or file system.This will enable an option to “Open in IDE” through browser extension on various websites like github, gitlab,gitbucket etc similar to one you see in this demo below.
```
Comment (maxandersen) [inline README.adoc:3]: I know I used this as an example before but "or start a debugger" is very specific and we might not do that as the first thing - I suggest to make it a bit more open ended, like "and other commands locally".

File: README.adoc [hunk 2]
```diff
@@ -1,50 +1,31 @@
...
-
-An outline of the idea with some screenshots/overview can be found https://docs.google.com/presentation/d/1AwWtUVz04b5u2I3QB-fEevzpyAJLSE1PzdS0JjnVoO0/edit#slide=id.pp[here] - feel free to comment.
-
-- Skill level: Beginner/Intermediate
-- Contact(s) / potential mentors(s): Max Rydahl Andersen (manderse@redhat.com)
-- Associated JBoss community project(s): Quarkus, vscode extensions, JBoss Tools
+## Targeted Platforms:
+- Linux
+- Windows
+- MAC OSX
```
Comment (maxandersen) [inline README.adoc:28]: minor: its macOS, OS X or Mac OS X :)

Comment (fahad-israr) [reply README.adoc:28]: a blunder....will do the changes

**Comment by maxandersen** [comment]
looks good!

**Pull request merged**